doc/external_resources.md

```diff
@@ -0,0 +1,8 @@
+# External resources
```

Comment:
Files like these are required to create the navigation in Sphinx. They aren't included in the current format.